cheroot/test/webtest.py

```diff
@@ -313,7 +213,16 @@ def interactive(self):
         False or 1 or 0.
         """
         env_str = os.environ.get('WEBTEST_INTERACTIVE', 'True')
-        return bool(json.loads(env_str.lower()))
+        is_interactive = bool(json.loads(env_str.lower()))
+        if is_interactive:
+            warnings.warn(
+                'We are moving to pytest-based testing and thus deprecating '
+                'WEBTEST_INTERACTIVE environment variable support. '
+                'Interactive test failure interceptor is going to be removed '
+                'in Cheroot v7.0.0',  # TODO: decide whether it's v7 or v8
```

Comment:
I've learned that it's difficult to predict when a feature will be dropped unless you have a project with long iterations and scheduled releases, neither of which apply to cheroot. I would simply say "WEBTEST_INTERACTIVE is deprecated." ...and in a future, backward-incompatible release, which we will determine when we get to it, we drop support.